unity-renderer/Assets/DCLPlugins/BuilderInWorld/HUD/ProjectsPanelHUD/Scripts/NewProject/NewProjectFlowController.cs

```diff
@@ -76,6 +78,13 @@ public bool IsActive()
 
     public void NewProject()
     {
+        UserProfile userProfile = UserProfile.GetOwnUserProfile();
+        if (string.IsNullOrEmpty(userProfile.ethAddress))
```

Comment:
Instead of checking if the eth address is null isn't it better if we add a function to UserProfile called like isGuestUser so that if the logic of being guest changes in the future it's updated everywhere?

Reply:
Resolve in https://github.com/decentraland/unity-renderer/pull/1840